HousingRepairsSchedulingApi.Tests/GatewaysTests/AppointmentsGatewaysTests.cs

```diff
@@ -0,0 +1,29 @@
+using System.Net;
```

Comment:
As this isn't currently used and not adding value, I think it should be deleted and can be created once an actual `IAppointmentsGateway` is created.